sparklightautoml/ml_algo/boost_lgbm.py

```diff
@@ -490,9 +466,9 @@ def fit(self, train_valid: SparkBaseTrainValidIterator):
     def get_features_score(self) -> Series:
         imp = 0
         for model_feature_impotances in self._models_feature_importances:
-            imp = imp + pd.Series(model_feature_impotances)
+            imp += pd.Series(model_feature_impotances)
```

Comment:
i suppose you can use python list comprehension or direct applying numpy functions

Reply:
fixed